anvil/src/udev.rs

```diff
@@ -323,6 +285,89 @@ pub fn run_udev(log: Logger) {
         state.device_added(&mut display, dev, path.into())
     }
 
+    if let Ok(instance) = Instance::new(Version::VERSION_1_2, None, log.clone()) {
+        if let Some(physical_device) = PhysicalDevice::enumerate(&instance).ok().and_then(|devices| {
+            devices
+                .filter(|phd| {
+                    phd.has_device_extension(unsafe {
+                        CStr::from_bytes_with_nul_unchecked(b"VK_EXT_physical_device_drm\0")
```

Comment:
You should be able to use `ExtPhysicalDeviceDrmFn::name` from ash instead of manually creating that string.